python/tach/__init__.py

```diff
@@ -1,5 +1,5 @@
 from __future__ import annotations
 
-__version__: str = "0.28.1"
```

Comment:
This is the one that backs `tach --version`

Reply:
I see, thanks! :)